File: personal-portfolio-tracker-frontend/src/dtos/Request/AddCapitalChangeRequestDto.js
export default class AddCapitalChangeRequestDto {
  constructor(positionId, entitlementDate, type, changeInNoOfShares, note) {
    this.positionId = positionId;
    this.entitlementDate = entitlementDate;
    this.type = type;
    this.changeInNoOfShares = changeInNoOfShares;
    this.note = note;
  }
};

File: PersonalPortfolioTrackerBackend/PersonalPortfolioTracker/DTOs/Request/AddCapitalChangeRequestDto.cs
using System.Text.Json.Serialization;

namespace PersonalPortfolioTracker.DTOs.Request
{
    public class AddCapitalChangeRequestDto
    {
        [JsonPropertyName("positionId")]
        public int PositionId { get; set; }

        [JsonPropertyName("entitlementDate")]
        public string? EntitlementDate { get; set; }

        [JsonPropertyName("type")]
        public string? Type { get; set; }

        [JsonPropertyName("changeInNoOfShares")]
        public int ChangeInNoOfShares { get; set; }

        [JsonPropertyName("note")]
        public string? Note { get; set; }
    }
}